keras_nlp/models/gpt_neo_x/gpt_neo_x_decoder.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_nlp.models.gpt_neo_x.gpt_neo_x_attention import GPTNeoXAttention
+from keras_nlp.utils.keras_utils import clone_initializer
+
+from keras_nlp.layers.transformer_layer_utils import (  # isort:skip
```

Comment:
why all the isort skip here? can't we just import this normally?

Reply:
I did same as we did in TransformerDecoder layer

Reply:
yeah, we should clean up there too (not on this PR though). just make this two normal imports, and let black/isort do it's thing